### PR TITLE
Solgov tweaks

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
This adds centcomm access to solgov roles.
Adds a sechud icon for solgov roles.
Adds berets for solgov lieutenants.
Tweaks solgov lieutenant uniform to more clearly show their rank on cuffs.
Adds an specops variation of solgov marines (akin to gamma ERT) that has a grey outfit.
Tweaks solgov loadouts to make more sense:

- Regular marines get bulldog shotguns, specops marines get ARGs
- All solgov marine roles have standard NV goggles, some sort of magboot, a survival injector (like miners get)
- Implants added for all SolGov roles: Mindshields & death alarms for ERT and representative roles, sechud implants, flash, and nutriment for regular marines. Sechud, nutriment, hardened CNS, flash, and shielded eyes for specops
- Marines (not LTs) get assault belts loaded with ammo for their primary weapon and sidearm (m1911)
- LTs keep the Deagle, and get more ammo in their bag
- LTs get a handcuff box
- LTs get a stunbaton on their belt slot
- Marines get a whetstone

ARG mag sprite added as a subtype of 556 ammo (no more jamming toploader mags into a bottom loading gun)
Thanks to @warior4356 for the ARG mag sprites.

## Why It's Good For The Game
When admins spawn in solgov teams currently, they get stuck behind all off-station doors and admins have to delete the doors, this fixes that.
This also makes lieutenants easier to distinguish, a better experience for everyone.
An "elite" variant outfit means its much quicker for admins to run a more advanced team without setup.
NT are leasing space from SolGov, it would make sense for their officers and representatives to show up on sechuds.

## Images of changes
image

imageimage

## Changelog
:cl:
add: sechud icon for solgov
add: solgov lieutenant beret
add: solgov specops marine and lieutenant admin outfit
tweak: ARG rifle now uses a bottom-loading magazine
tweak: adds CC access to solgov ERT
tweak: solgov lieutenant uniforms more clearly show rank
/:cl:

### CODE REVIEW

**Review by E-MonaRhg — COMMENTED** [review]
Looks great but a couple things and my first ever code review, yay.

1. I think that the ARG is better than the bulldog, maybe swap these around
2. And here comes my review:

File: code/modules/clothing/head/soft_caps.dm [hunk 1]
```diff
@@ -123,10 +123,26 @@
 	icon_state = "solgovsoft"
 	item_color = "solgov"
 	dog_fashion = null
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 50)
```
[the remark this answers is not in the document]
Comment (S34NW) [reply code/modules/clothing/head/soft_caps.dm:126]: oh! I forgot this was a loadout item, ah nuts. I'll make a "military" version that just has different stats and used only by the marines

**Comment by S34NW** [comment]
> 
> 
> Looks great but a couple things and my first ever code review, yay.
> 
>     1. I think that the ARG is better than the bulldog, maybe swap these around
> 
>     2. And here comes my review:

Good points, changes made.

File: code/datums/outfits/outfit_admin.dm [hunk 2]
```diff
@@ -634,65 +634,120 @@
...
+		/obj/item/reagent_containers/hypospray/autoinjector/survival = 1,
+		/obj/item/ammo_box/magazine/m50 = 3
+	)
+	is_solgov_lieutenant = TRUE
+
+/datum/outfit/admin/solgov/elite
+	name = "Solar Federation Specops Marine"
+	uniform = /obj/item/clothing/under/solgov/elite
+	head = /obj/item/clothing/head/soft/solgov/marines/elite
+	belt =/obj/item/storage/belt/military/assault/marines/elite/full
```
Comment (S34NW) [inline code/datums/outfits/outfit_admin.dm:717]:
```suggestion
	belt = /obj/item/storage/belt/military/assault/marines/elite/full
```
oops

**Comment by FlattestGuitar** [comment]
You could maybe take the opportunity to add some shading to the uniforms to make them more in line with what uniforms look like in general.

When I originally made that sprite I couldn't sprite very well. 

I still can't sprite very well.

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**